sections/04_study.md

```diff
@@ -105,7 +105,81 @@ learning can address is poised to improve.
+factors. This remarkable complexity lends itself to defects that underlie many
+diseases [@tag:Scotti2016_missplicing]. For instance, in Becker muscular
+dystrophy, a point mutation in dystrophin creates an ESS that induces skipping
+of exon 31. A recent study found that quantitative trait loci (QTLs) that affect
+splicing in lymphoblastoid cell lines are enriched within risk loci for
+schizophrenia, multiple sclerosis and other immune diseases, implicating
+mis-splicing as a much more widespread feature of human pathologies than
+previously thought [@tag:Li2016_variation].
+
+Sequencing studies routinely return thousands of unannotated variants. Which
+cause functional changes in splicing, and if so, how? Prediction of a “splicing
+code” has been a holy grail over the past decade. Initial machine learning
+approaches used a naive Bayes model and a 2-layer Bayesian neural network with
+thousands of hand-derived sequence-based features to predict the probability of
+exon skipping [@tag:Barash2010_splicing_code @tag:Xiong2011_bayesian]. With the
+advent of deep learning, more complex models were built that provided
+significantly better performance [@tag:Xiong2015_splicing_code
+@tag:Jha2017_integrative_models]. Importantly, these new approaches can take in
+not only genomic features, but also tissue identity and CLIP-seq measurements of
+interactions between splicing factors and RNA, which all improve predictive
+accuracy.
+
+The massive improvement seen with deep learning seems to stem from hidden layers
+being able to create new higher-order “features”, whereas earlier approaches
+often assumed independence of features and were unable to generalize.
+Higher-order understanding is especially important in splicing, which depends
+not only on the primary sequence, but also local RNA structure, tissue identity,
+splicing factor binding, and other currently unknown factors — all of which
+interact in complex, incompletely characterized ways. With new tools to
+interpret these meta-features, a major focus of current deep learning research,
+we will soon have the ability to extract a more nuanced biological understanding
+of splicing — perhaps by building different deep neural networks for different
+biological conditions, then interrogating their hidden nodes.
+
+A parallel effort has been to use more data with simpler models. An exhaustive
+study using readouts of splicing for millions of synthetic intronic sequences
+was able to describe motifs that influence the strength of alternative splice
+sites [@tag:Rosenberg2015_synthetic_seqs]. Interestingly, a simple linear model
+built from this data successfully generalized to exon skipping: in a limited
+analysis using SNPs from three genes, it predicted exon skipping with three
+times the accuracy of Xiong et al.’s framework. This case is instructive in that
+clever sources of data, not just more powerful models, can lead to novel
+insights.
```

Comment:
I feel like it's both. Different/complementary kinds of data provide opportunities to build more informative models, and better algorithms let you do more with integrating more diverse sources of data. I don't know if anyone has a definite answer on this or whether there should be a definite answer..